modules/puppetca_hostverify/puppetca_hostverify_autosigner.rb

```diff
@@ -0,0 +1,57 @@
+module ::Proxy::PuppetCa::Hostverify
+  class Autosigner
```

Comment:
Sorry to be so picky about the names, but what about `FileBasedAutosign`.

Reply:
The token-based-autosigning-approach will also use a file, so that would not work. See above.